README.md

```diff
@@ -0,0 +1,46 @@
+pypi
+====
+
+sentry internal pypi
+
+this repository contains the tools to import and/or build packages from public pypi for the
```

Comment:
If we placed all keyboards of all engineers in the world side by side I would still recognize Anthony's brand-new-shift-key's keyboard! 🤣

Reply:
lol